extensions/sql-migration/src/api/azure.ts

```diff
@@ -166,12 +166,24 @@ export async function startDatabaseMigration(account: azdata.Account, subscripti
 		throw new Error(response.errors.toString());
 	}
 	return {
-		errors: response.errors,
 		status: response.response.status,
 		databaseMigration: response.response.data
 	};
 }
 
+export async function getMigrationStatus(account: azdata.Account, subscription: Subscription, migration: DatabaseMigration): Promise<any> {
```

Comment:
You should have an actual return type here - not `any`

Reply:
I will get around that in the next iteration. When I actually implement the migration status dialog

Reply:
Is there any reason why you wouldn't want to do it now? Even if it's not the final form having something for the type will help ensure that everything is hooked up correctly.